src/NLog.StructuredLogging.Json/LoggerExtensions.cs

```diff
@@ -35,7 +35,7 @@ public static void ExtendedError(this ILogger logger, string message, object log
         }
 
         public static void Extended(this ILogger logger, LogLevel logLevel, string message, object logProperties,
-            Exception ex = null)
+            Exception ex = null, string tag = "")
```

Comment:
@martincostello do you want the tag to be empty by default? Currently, it's a new guid by default.